src/grpc-metric-exporter.ts

```diff
@@ -0,0 +1,34 @@
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-grpc';
+import {
+  DATASET_HEADER_KEY,
+  OTLP_HEADER_KEY,
+  OTLP_PROTO_VERSION,
+  TEAM_HEADER_KEY,
+} from './exporter-utils';
+import {
+  computeOptions,
+  HoneycombOptions,
+  isClassic,
+} from './honeycomb-options';
```

Comment:
```suggestion
import { computeOptions, HoneycombOptions } from './honeycomb-options';
```

`isClassic` isn't needed in this one, prettier then smushes it to one line